crates/walrus-service/storage_openapi.yaml

```diff
@@ -218,7 +224,7 @@ paths:
               schema:
                 $ref: '#/components/schemas/ApiSuccess_String'
         '400':
-          description: May be returned when (1) Storing the metadata cannot be completed because the blob has been marked as invalid by the system. (2) The blob has not been registered or has already expired. (3) The provided metadata is not valid for the blob.
+          description: May be returned when (1) Storing the metadata cannot be completed because the blob has been marked as invalid by the system. (2) The blob has not been registered or has already expired. (3) The node refused the metadata upload because the pending cache is full. (4) The provided metadata is not valid for the blob.
```

Comment:
Nit (preexisting): We should probably differentiate the description of 400 errors for the different endpoints.

Reply:
fixed